.gitignore

```diff
@@ -0,0 +1,3 @@
+builddir/
+.vscode/
+build/
```

Comment:
See https://github.com/libratbag/libratbag/pull/671 or https://github.com/libratbag/libratbag/pull/871.

Reply:
Removed, makes sense. I setup my globals, especially with the way Meson and Ninja work.